packages/discord.js/src/structures/interfaces/InteractionResponses.js

```diff
@@ -260,6 +260,22 @@ class InteractionResponses {
     this.replied = true;
   }
 
+  /**
+   * Responds to the interaction with an upgrade button.
+   * <warn>Only available for applications with monetization enabled.</warn>
+   * @returns {Promise<void>}
+   */
+  async sendPremiumUpgradeButton() {
+    if (this.deferred || this.replied) throw new DiscordjsError(ErrorCodes.InteractionAlreadyReplied);
```

Comment:
Can you not defer and then send this? really?? or do you need to edit it in the reply?